components/tidb_query_common/src/string.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Maybe the module `tidb_query_shared_expr` is an appropriate location to place.

Reply:
@zhongzc Comment addressed. PTAL~